driver/mounter.go

```diff
@@ -140,7 +153,20 @@ func (m *mounter) Mount(source, target, fsType string, opts ...string) error {
 		mountArgs = append(mountArgs, "-o", strings.Join(opts, ","))
 	}
 
-	mountArgs = append(mountArgs, source)
+	if !isBind && luksContext.EncryptionEnabled {
```

Comment:
I was trying to avoid adding `isBind` to the function signature, but it is a bit complicated to figure out if the current mount is happening during the staging or publishing phase.

Instead of doing a complicated detection that might be prone to errors, I decided to just add the `isBind` parameter to the function, because it makes it very explicit.

I'll take another look at it. If I can derive the current stage without adding too much complex logic, I'll remove it.

If that doesn't work, I would remove `isBind` from the function signature and pass it along implicitly in the `LuksContext` struct, e.g. with a `VolumeLifecycle` element. This might come in handy in the future.

Reply:
I replaced the boolean by passing the current lifecycle phase into the `LuksContext`. This is less complex and error-prone than inspecting the volume / filesystem on the node to figure out if this is the staging phase or not. Please let me know what you think.